### PR TITLE
Add ruby version in Gemfile

### PR DESCRIPTION
**Pull request by Hanzolskl**
in gemfile add line 
`ruby '2.3.1'` of course if use 2,3,1

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @post = Post.new
+    respond_with(@post)
+  end
+
+  def edit
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.save
```
Comment (Magons) [inline app/controllers/posts_controller.rb]: better use create instead new and save

File: app/models/user.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
+  has_many :post
```
Comment (Magons) [inline app/models/user.rb]: not post, posts

File: app/views/layouts/mailer.html.haml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
+    :css
```
Comment (Magons) [inline app/views/layouts/mailer.html.haml]: what is this?

File: test/controllers/posts_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  test "should get new" do
+    get new_post_url
+    assert_response :success
+  end
+
+  test "should create post" do
+    assert_difference('Post.count') do
+      post posts_url, params: { post: { User_id: @post.User_id, body: @post.body, title: @post.title } }
```
Comment (Magons) [inline test/controllers/posts_controller_test.rb]: User_id -> user_id

File: test/controllers/posts_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get edit_post_url(@post)
+    assert_response :success
+  end
+
+  test "should update post" do
+    patch post_url(@post), params: { post: { User_id: @post.User_id, body: @post.body, title: @post.title } }
```
Comment (Magons) [inline test/controllers/posts_controller_test.rb]: the same

**Comment by Hanzolskl** [comment]
W8 your check and new issues for me.

**Pull request merged**